src/interfaces/BulkUpload.ts

```diff
@@ -0,0 +1,7 @@
+export interface BulkUpload {
+  id: string;
+  fileName: string;
+  fileSize: string;
```

Comment:
Hm, it looks like we are not storing the file size in the database at the moment, so the API is not returning it. That seems like something we could reasonably cache, but maybe not in this first iteration? cc @slifty

Reply:
Got it -- I'd vote we make an issue for it so we can implement it the Right Way after the demo.

Reply:
https://github.com/PhilanthropyDataCommons/service/issues/651